run_nextflow_RNAseq.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+workflow_path='/hpc/diaggen/users/lonneke/github/DxNextflowRNA'
+
+# Set input and output dirs
+input=`realpath $1`
+output=`realpath $2`
+email=$3
+
+mkdir -p $output && cd $output
+mkdir -p log
+
+if ! { [ -f 'workflow.running' ] || [ -f 'workflow.done' ] || [ -f 'workflow.failed' ]; }; then
+touch workflow.running
+echo "check directory for output: ${output}"
+
+export JAVA_HOME='/hpc/diaggen/software/tools/jdk-18.0.2.1/'  # change java version 
+export NXF_JAVA_HOME='/hpc/diaggen/software/tools/jdk-18.0.2.1/'  # change java vesion of nextflow
+
+sbatch <<EOT
+#!/bin/bash
+#SBATCH --time=04:00:00
+#SBATCH --nodes=1
+#SBATCH --mem 10G
+#SBATCH --gres=tmpspace:10G
+#SBATCH --job-name Nextflow_RNASeq
+#SBATCH -o log/slurm_nextflow_rnaseq.%j.out
+#SBATCH -e log/slurm_nextflow_rnaseq.%j.err
+#SBATCH --mail-user $email
+#SBATCH --mail-type FAIL
+#SBATCH --account=diaggen
+
+/hpc/diaggen/software/development/DxNextflowRNA/tools/nextflow run /hpc/diaggen/users/lonneke/github/DxNextflowRNA/main.nf  \
+-c /hpc/diaggen/users/lonneke/github/DxNextflowRNA/nextflow.config -resume -ansi-log false -profile slurm \
```

Comment:
- Could you change your user folder paths to `/hpc/diaggen/software/development/`?
- Can you split this into multiple lines, where each variable is on a single line?